ml-agents-envs/mlagents_envs/base_env.py

```diff
@@ -109,6 +109,9 @@ def agent_id_to_index(self) -> Dict[AgentId, int]:
     def contains_agent(self, agent_id: AgentId) -> bool:
         return agent_id in self.agent_id_to_index
 
+    def get_index(self, agent_id: AgentId) -> int:
```

Comment:
Why do we need this function rather than just using `agent_id_to_index[agent_id]?` Since you're iterating over the list of agents it's guaranteed to exist